### PR TITLE
Test transactionQueue module

### PR DESCRIPTION
**Pull request by nicoayala**
### What does this Pull Request do?
Adds tests to the transactionQueue module.

### Are there points in the code the reviewer needs to double check? 
* Logger configuration for test environment is out of scope, but it is handy when it comes to debugging your tests (it only logs to a file).
* Fix a bug regarding the mocking of `createLevelStore`: if no database name was supplied the mocked leveldb would fail in specific cases.

### Does this Pull Request meet the acceptance criteria?
- [x] I have read the [Contribution guidelines](https://github.com/wibsonorg/buyer-sdk/CONTRIBUTING.md).
- [x] I have read the [Code of Conduct](https://github.com/wibsonorg/buyer-sdk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: buyer-api/src/facades/helpers/performTransaction.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { delay } from '../../utils';
+import { delay } from '../../utils/delay';
```
Comment (Agus) [inline buyer-api/src/facades/helpers/performTransaction.js]: Why should we use the absolute path and not the one in index?

**Comment by Agus** [comment]
Since you have now a `buyer-api/test/support/` folder, what if you have all the web3/blockchain related mock stuff there? For example having an exported fakeReceipt there. I guess that this kind of objects will be used in lot of places.
https://github.com/wibsonorg/buyer-sdk/blob/ce28e651d18ab350ff8db17893b0ce4e4343c4ab/buyer-api/test/queues/transactionQueue.test.js#L10-L19

File: buyer-api/src/facades/helpers/performTransaction.js (hunk 1 above)
Comment (nicoayala) [reply buyer-api/src/facades/helpers/performTransaction.js]: Yes, it makes no sense, I'll revert it!

**Comment by nicoayala** [comment]
@Agus @danielfx90 I found some bugs when testing against Ropsten. This is now ready for review.

**Comment by nicoayala** [comment]
@Agus You are right! The truthiness of `_blockNumber` takes precedence over the assignment, that is why it always returns 0. Also, the assignment also returns the new value. Worst line of code so far, I'm pushing my limits!

**Pull request merged**